src/rabbit-lru-cache.ts

```diff
@@ -114,12 +133,26 @@ export async function createRabbitLRUCache<T>(options: RabbitLRUCacheOptions<T>)
         return channel;
     }
 
-    async function handleConnectionError(error: Error, attempt = 0, retryInterval = 0): Promise<void> {
+    function getRetryInterval(attempt: number, accumulated: number): number {
+        const { retryIntervalUpTo, retryBase, retryMethod } = reconnectionOptions;
```

Comment:
As discussed let's simplify and just use retryMaxInterval, retryInterval, retryFactor.

Reply:
Agree, i think this looks much better. Changed to match those parameters and retry algorithm.